### PR TITLE
Bugfix/es settings

### LINKED ISSUE #1251: Index settings incorrect
Running an index with the `-p` flag will incorrectly create an index with 1 shard and 1 replica (should be: 5 / 1). Should be fixed, *especially* before running a big indexing job from source. @Meesch

### PR DESCRIPTION
**Pull request by BeritJanssen**
close #1251

### CODE REVIEW

File: functional-tests/latest_logs/test_routing.test_filter/basic_test_info.txt
Comment (lukavdplas) [inline functional-tests/latest_logs/test_routing.test_filter/basic_test_info.txt:1]: This (and everything else from the directory) looks like it have been in gitignore.

File: backend/es/es_index.py [hunk 1]
```diff
@@ -44,8 +44,7 @@ def create(client, corpus_definition, add, clear, prod):
             sys.exit(1)
 
         logger.info('Adding prod settings to index')
-        index_settings = settings.get('index', {})
-        index_settings.update({
+        settings['index'].update({
             'number_of_replicas' : 0,
```
Comment (lukavdplas) [inline backend/es/es_index.py:48]: Shouldn't we use `number_of_replicas: 1` in production? 🤔

**Review by lukavdplas — CHANGES_REQUESTED** [review]
Ignoring the functional test output, this looks good!

**Comment by BeritJanssen** [comment]
Ah, I thought I'd been careful not to commit those files, but seems I hit the "commit all" button at some point.

File: backend/es/es_index.py (hunk 1 above)
Comment (BeritJanssen) [reply backend/es/es_index.py:48]: Only after the indexing procedure is complete. That happens at the end of the indexing script.

**Review by lukavdplas — APPROVED** [review]

**Pull request merged**